test/fw/ptl/utils/pbs_snaputils.py

```diff
@@ -950,19 +295,36 @@ class _PBSSnapUtils(object):
     PBS snapshot utilities
     """
 
-    def __init__(self, snap_args, create_tar=False):
+    def __init__(self, out_dir, diag_path=None, server_host=None,
+                 log_level=None, acct_logs=None, srvc_logs=None,
+                 additional_hosts=None, map_file=None, anonymize=None,
+                 create_tar=False):
         """
         Initialize a PBSSnapUtils object with the arguments specified
 
-        :param snap_args: A dict of arguments/options accepted by pbs_snapshot.
-        "out_dir" is the only mandatory item needed.
-        :type snap_args: dict
+        :param out_dir: name of the output snapshot
+        :type out_dir: String
+        :param diag_path: Path to the diag directory to create snapshot from
+        :type diag_path: String or Nonetype
```

Comment:
Well since I was just commenting about consistency, I'll leave it up to @hirenvadalia to comment.